tests/compile-fail/data_race/dealloc_read_race_stack.rs

```diff
@@ -27,9 +28,6 @@ pub fn main() {
     //  3. stack-deallocate
     unsafe {
         let j1 = spawn(move || {
-            // Concurrent allocate the memory.
-            // Uses relaxed semantics to not generate
-            // a release sequence.
```

Comment:
@JCTyblaidd these comments seem to be wrong here; there is no allocation and also nothing `relaxed` anywhere nearby...